config/routes.rb

```diff
@@ -2,7 +2,9 @@
 
   devise_for :users, controllers: { omniauth_callbacks: 'omniauth' }
 
-  resources :users
+  resources :users do
+    get :tickets, to: 'tickets#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.